.circleci/config.yml

```diff
@@ -85,6 +85,10 @@ references:
       - deploy:
           name: Install Codacy
           command: make -C .doks/ deploy_to_doks
+      - deploy:
+          name: Update codacy-api ip address
+          command: |
+            ip=$(kubectl get services codacy-api --namespace codacy-nightly | awk '{print $4}' | tail -n +2); helm upgrade $RELEASE_NAME --set codacy-api.url=$ip
```

Comment:
I think its safer if you use `--jsonpath` here to get the ip for the service.

Reply:
good call :-) i have replaced the command in the next commit.